_includes/content.html

```diff
@@ -94,28 +105,22 @@
 {% highlight ruby %}
 Griddler.configure do |config|
   config.processor_class = EmailProcessor # MyEmailProcessor
-  config.to = :token # :full, :email, :hash
-  # :raw    => 'AppName <s13.6b2d13dc6a1d33db7644@mail.myapp.com>'
-  # :email  => 's13.6b2d13dc6a1d33db7644@mail.myapp.com'
-  # :token  => 's13.6b2d13dc6a1d33db7644'
-  # :hash   => { raw: '', email: '', token: '', host: '' }
+  config.processor_method = :process
```

Comment:
The UL/LI block on the page talks about the processor method, but its not actually shown in the highlighted Ruby code!